test/WebSites/ControllersFromServicesWebSite/Startup.cs

```diff
@@ -16,34 +16,37 @@ namespace ControllersFromServicesWebSite
 {
     public class Startup
     {
-        public void Configure(IApplicationBuilder app)
+        // Set up application services
+        public IServiceProvider ConfigureServices(IServiceCollection services)
         {
-            var configuration = app.GetTestConfiguration();
-
-            app.UseServices(services =>
-            {
-                services.AddMvc()
-                        .WithControllersAsServices(
-                         new[]
-                         {
+            services.AddMvc()
+                    .WithControllersAsServices(
+                     new[]
+                     {
                             typeof(TimeScheduleController).GetTypeInfo().Assembly
-                         });
+                     });
```

Comment:
fyi no need to line up the periods; just hit enter after `services` or `services.AddMvc()` and VS should do the Right Thing:tm: